packages/ui/src/routes/post/new/+page.svelte

```diff
@@ -84,6 +84,14 @@
 		align-items: center;
 		font-weight: 600;
 		padding: var(--spacing-12) var(--spacing-12) var(--spacing-24);
+		max-width: 1280px;
```

Comment:
Is this correct? I the maximum 1280px? I thought we have wider pages as well.

Reply:
you know, you might be right. I remember David talking about wider grids in an earlier call but I don't remember what the final decision was at that point. Currently, the timeline grid does have a max of 1280px so I'd keep the headers consistent with that, and when I implement the new design/grid I can adjust the headers to match that as well. Thoughts?

Reply:
Ok. Suggestion - make it a variable defined in global styles and change it based on media query :). I think even now it should be css variable as it is used in few places.

Reply:
Ah yes, will do 😊